test/powershell/Language/Scripting/NativeExecution/NativeStreams.Tests.ps1

```diff
@@ -49,14 +49,25 @@ Describe "Native streams behavior with PowerShell" -Tags 'CI' {
         It 'preserves error stream as is with Out-String' {
             ($out | Out-String).Replace("`r", '') | Should Be "foo`n`nbar`n`nbazmiddlefoo`n`nbar`n`nbaz`n"
         }
+
+        It 'does not get truncated or split when redirected' {
+            $longtext = "0123456789"
+            while ($longtext.Length -lt [console]::WindowWidth) {
+                $longtext += $longtext
+            }
+            pwsh -c "& { [Console]::Error.WriteLine('$longtext') }" 2>&1 > $testdrive\error.txt
+            $e = Get-Content -Path $testdrive\error.txt
+            $e.Count | Should BeExactly 1
+            $e | Should Match $longtext
```

Comment:
Maybe use "BeExactly"? It seems better for testing a formatting.

Reply:
The problem currently is that redirection writes with a BOM, so this would fail

Reply:
We read with `Get-Content -Path $testdrive\error.txt` - I believe there is not BOM if we don't use `-Byte`.

Reply:
Good point about Get-Content, I'll change it